docs/docs-content/clusters/public-cloud/azure/required-permissions.md

```diff
@@ -129,172 +114,136 @@ dynamically creates the network resources required for your cluster.
+       "Microsoft.Compute/images/write",
+       "Microsoft.Compute/virtualMachines/delete",
+       "Microsoft.Compute/virtualMachines/extensions/delete",
+       "Microsoft.Compute/virtualMachines/extensions/read",
+       "Microsoft.Compute/virtualMachines/extensions/write",
+       "Microsoft.Compute/virtualMachines/read",
+       "Microsoft.Compute/virtualMachines/write",
+       "Microsoft.Network/loadBalancers/backendAddressPools/join/action",
+       "Microsoft.Network/loadBalancers/delete",
+       "Microsoft.Network/loadBalancers/inboundNatRules/delete",
+       "Microsoft.Network/loadBalancers/inboundNatRules/join/action",
+       "Microsoft.Network/loadBalancers/inboundNatRules/read",
+       "Microsoft.Network/loadBalancers/inboundNatRules/write",
+       "Microsoft.Network/loadBalancers/read",
+       "Microsoft.Network/loadBalancers/write",
+       "Microsoft.Network/networkInterfaces/delete",
+       "Microsoft.Network/networkInterfaces/join/action",
+       "Microsoft.Network/networkInterfaces/read",
+       "Microsoft.Network/networkInterfaces/write",
+       "Microsoft.Network/publicIPAddresses/delete",
+       "Microsoft.Network/publicIPAddresses/join/action",
+       "Microsoft.Network/publicIPAddresses/read",
+       "Microsoft.Network/publicIPAddresses/write",
+       "Microsoft.Resources/subscriptions/resourceGroups/read",
+       "Microsoft.Resources/subscriptions/resourceGroups/write",
+       "Microsoft.Storage/storageAccounts/blobServices/containers/read",
+       "Microsoft.Storage/storageAccounts/blobServices/containers/write",
+       "Microsoft.Storage/storageAccounts/listKeys/action",
+       "Microsoft.Storage/storageAccounts/read",
+       "Microsoft.Storage/storageAccounts/write"
+     ],
+     "NotActions": [],
+     "AssignableScopes": [
+       "/subscriptions/$SUBSCRIPTION_ID"
+     ]
+   }
+   ```
+
+4. Create a role for each of the JSON files you created in the previous steps. Issue the following commands to create
+   the roles.
+
+   ```shell
+   az role definition create --role-definition @iaas_static_vnet_role.template.json --output table
+   az role definition create --role-definition @iaas_static_rg_sub_role.template.json --output table
+   ```
+
+5. Export the
+   [security principal](https://learn.microsoft.com/en-us/azure/role-based-access-control/overview#security-principal)
+   object ID you want to use with Palette to a variable.
+
+   ```shell
+   export ASSIGNEE="<security_principal_object_id>"
+   ```
+
+6. Assign the roles to the service principal. Use the following commands to assign the roles.
+
+   ```json
+   az role assignment create --assignee $ASSIGNEE \
+     --role "Palette Static Placement IaaS Cluster Deployer (rg/sub)" \
+     --scope "/subscriptions/$SUBSCRIPTION_ID"
+   ```
+
+   ```json
+   az role assignment create --assignee $ASSIGNEE \
+     --role "Palette Static Placement IaaS Cluster Deployer (vnet)" \
+     --scope "/subscriptions/$SUBSCRIPTION_ID"
```

Comment:
This doesn't look right. The roles have scope indicated in the end of their name ("vnet" here). This role should be applied at the virtual network scope level, not subscription.